src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -138,6 +138,9 @@ private function getContainerCompilerLogs()
         $logs = array();
         foreach (file($file, FILE_IGNORE_NEW_LINES) as $log) {
             $log = explode(': ', $log, 2);
+            if (!isset($log[1]) || !class_exists($log[0])) {
```

Comment:
let's do it with a regexp instead of class_exists?
`|| !preg_match('/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*+(?:\\\\[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*+)++$/', $log[0])`